Automattic/jetpack
# E2E Tests: Try localtunnel instead of ngrok

#### Changes proposed in this Pull Request:

* replace ngrok with self-hosted localtunnel

#### Jetpack product discussion


n/a
#### Does this pull request change what data or activity we track or use?


n/a
#### Testing instructions:






* Run tests locally:
- `yarn build`
- `./tests/e2e/bin/env.sh start`
- `PUPPETEER_HEADLESS=false yarn test-e2e --testPathIgnorePatterns=update`
* Make sure E2Es are passing in CI

#### Proposed changelog entry for your changes:

* n/a
